src/lib/components/SensorDescription.svelte

```diff
@@ -40,7 +64,21 @@
 		<CollapsibleParagraph linesClampedCount={3} className="text-sm">
 			<p>{description}</p>
 			<strong class="mb-1 mt-4 block text-base font-semibold">
-				{$LL.pages.stations.stationsDescriptions.supportedIndicatorsLabel()}:
+				{$LL.pages.stations.stationsDescriptions.measuredIndicatorsLabel()}:
+			</strong>
+			<ul class="flex flex-col gap-1 pb-4">
+				{#each measuredIndicatorsList as { value, label, unitOnly } (value)}
+					<li class="border-b border-muted pb-1">
+						{label}
+						{#if unitOnly}
+							<span class="text-muted-foreground"> ({unitOnly})</span>
+						{/if}
+					</li>
+				{/each}
+			</ul>
+			<p>{descriptionCalculated}</p>
+			<strong class="mb-1 mt-4 block text-base font-semibold">
+				{$LL.pages.stations.stationsDescriptions.calculatedIndicatorsLabel()}:
```

Comment:
This will result in this:

![image](https://github.com/user-attachments/assets/5e8766db-8bd9-4e2d-8696-80e8399e0176)

Reply:
> This will result in this

Looking good! Thx for the initiative!